src/Compilers/Core/Portable/MetadataReader/TypeNameDecoder.cs

```diff
@@ -98,6 +98,11 @@ protected bool IsAcceptedInAttributeModifierType(TypeSymbol type)
             return _factory.IsAcceptedInAttributeModifierType(type);
         }
 
+        protected bool IsAcceptedUnmanagedTypeModifierType(TypeSymbol type)
```

Comment:
>protected bool IsAcceptedUnmanagedTypeModifierType(TypeSymbol type) [](start = 8, length = 67)

It doesn't look like this method is used anywhere.

Reply:
Used in MetadataDecoder.cs

---
In reply to: [172347229](https://github.com/dotnet/roslyn/pull/24817#discussion_r172347229) [](ancestors = 172347229)